src/js/consts/UoMType.js

```diff
@@ -0,0 +1,5 @@
+// eslint-disable-next-line import/prefer-default-export
```

Comment:
how about doing:
```js
const UomType = {...}

export default UomType;
```
so that you wouldn't have to disable the eslint?

Reply:
Moreover I'm starting to think, that maybe we should align the naming with the backend's enum, so to use a full name: `UnitOfMeasureType`. What do you think?